axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -372,23 +430,93 @@ public TrackingToken createHeadToken() {
         public TrackingToken createTokenAt(Instant instant) {
             try {
                 io.axoniq.axonserver.grpc.event.TrackingToken token = eventStoreClient.getTokenAt(instant).get();
-                if( token.getToken() < 0 ) return null;
-                return new GlobalSequenceTrackingToken(token.getToken()-1);
+                if (token.getToken() < 0) {
+                    return null;
+                }
+                return new GlobalSequenceTrackingToken(token.getToken() - 1);
             } catch (Throwable e) {
                 throw ErrorCode.convert(e);
             }
         }
 
         @Override
         protected Stream<? extends TrackedEventData<?>> readEventData(TrackingToken trackingToken, boolean mayBlock) {
-            throw new UnsupportedOperationException("This method is not optimized for the AxonIQ Event Store and should not be used");
+            throw new UnsupportedOperationException(
+                    "This method is not optimized for the AxonIQ Event Store and should not be used"
+            );
         }
 
         @Override
         protected Stream<? extends DomainEventData<?>> readSnapshotData(String aggregateIdentifier) {
-            // snapshots are automatically fetched server-side, which is faster
+            // Snapshots are automatically fetched server-side, which is faster
             return Stream.empty();
         }
 
+        private static class Builder extends AbstractEventStorageEngine.Builder {
+
+            private AxonServerConfiguration configuration;
+            private AxonDBClient eventStoreClient;
```

Comment:
@MGathier / @saratry

Reply:
To keep this consistent, I think we need to change the class name. What do you think @MGathier ?

Reply:
I'll let you guys (@saratry / @MGathier) pick this up a latter time. Don't think this should block this PR.

Reply:
Yes, it should not block this PR

I discussed today with Frans the concept of DB/Hub. As I understand the name AxonDB will not live any longer. It will be the event store in Axon Server or something like that. So I am not sure at all how this will be represented in the rest of the code. But might be nice to do a clean up of this.